source-reminder/lambda_function.py

```diff
@@ -51,13 +54,35 @@ def request_data(limit, start_date, end_date):
     return 0
 
 
+def request_author(slug):
+    try:
+        response_data = requests.get(API_ENDPOINT + slug)
+        return response_data
+    except HTTPError as errh:
+        print("HTTP Error:", errh)  # Should Slack send a message to the channel, stating that an error occurred?
```

Comment:
@bfreeds, do you think sentry is overkill for reporting this and the error below?

Reply:
@ashley-hebler I think Sentry is a great idea and doesn't take too much to setup.  It's something that I think we will thank ourselves for later, and would be helpful for getting setup for other slackbots for more transparent error handling.

Reply:
@MatthewTurk247, this could be in a future PR and doesn't necessarily have to happen here, IMO, but wanted to leave a couple Sentry documentation resources for how to set it up with Lambda:
- [Sentry python aws lambda guide](https://docs.sentry.io/platforms/python/guides/aws-lambda/)
- [Sentry Python Lambda examples](https://github.com/getsentry/examples/tree/master/aws-lambda/python)
When we're ready to add Sentry error handling I can configure Sentry and provide a `DSN` (that'd be another environment variable set).

Reply:
Totally agree that it doesn't have to happen here then. We always have the lambda logs to help debug for the first few weeks. @MatthewTurk247, feel free to resolve this and add later! We can start a new Basecamp task for it when the time comes.